deployment/command-line-option.md

```diff
@@ -161,21 +161,55 @@ Options:
 Control `fluentd` process using [Signals](signals.md) or Windows Event.
 
 ```text
-Usage: fluent-ctl COMMAND PID
+Usage: fluent-ctl COMMAND PID_OR_SVCNAME
```

Comment:
```suggestion
Usage: fluent-ctl COMMAND [PID_OR_SVCNAME]
```

Adding `[ ... ]` is recommended for `PID_OR_SVCNAME` to indicate it's an omittable argument.

Here is the actual help string of `fleunt-ctl` on Windows:
```console
aho@SKYLAKE C:\Users\aho\Projects\fluentd>bundle exec fluent-ctl

Usage: C:/Users/aho/Projects/fluentd/vendor/bundle/ruby/2.7.0/bin/fluent-ctl COMMAND [PID_OR_SVCNAME]

Commands:
  shutdown
  restart
  flush
  reload
  dump

Error: Command isn't specified!
```

Reply:
Fixed in https://github.com/fluent/fluentd-docs-gitbook/pull/397/commits/dc0ddcfc33deb68577738093d0d519d23bf1c68f